src/rust/src/datatypes.rs

```diff
@@ -145,15 +145,15 @@ impl PlRDataType {
         name.try_into().map_err(savvy::Error::from)
     }
 
-    pub fn new_decimal(scale: Option<NumericScalar>, precision: Option<NumericScalar>) -> Result<Self> {
+    pub fn new_decimal(
+        scale: Option<NumericScalar>,
+        precision: Option<NumericScalar>,
+    ) -> Result<Self> {
         let precision = precision
             .map(<Wrap<usize>>::try_from)
             .transpose()?
             .map(|p| p.0);
-        let scale = scale
-            .map(<Wrap<usize>>::try_from)
-            .transpose()?
-            .map(|s| s.0);
```

Comment:
I was surprised that VS Code's Rust extension does not support "formatting only selections"......